### DISCUSSION TITLE
Qusar with CASL

### DISCUSSION THREAD

**Discussion by sKY02644** [quasarframework/quasar#10549] [General - Components / Directives / etc]
Can someone guide me through using casl with quasar for role and permission please.

**Comment by ontwikkelfabriek** [comment]
@sKY02644 

Here's some info:
https://casl.js.org/v5/en/package/casl-vue
https://www.npmjs.com/package/@casl/vue

btw nobody here will `guide` you on a `non` Quasar `specific` topic, without you doing some research yourself first and show what you have done and what culpits you have come across.

**Comment by sKY02644** [comment]
Thank you at @ontwikkelfabriek iv gone through these setup already and got some errors 

like 

`vue-router.esm-bundler.js?f6c4:3242 TypeError: Cannot read property 'relevantRuleFor' of undefined`

and im sure im not setting it up well in my boot file im using  Quasar2/Vue3

**Reply by sKY02644** [reply]
i was just hoping someone has implemented it recently so i could be guided on how to set it up in the boot file and use in both vue files and ts files

**Comment by ontwikkelfabriek** [comment]
@sKY02644 

> and im sure im not setting it up well in my boot file im using Quasar2/Vue3

If you create a codepen.io or setup a demo repo with Quasar project with CASL-vue, I or someone else could take a look at it.

**Reply by sKY02644** [reply]
alright will do just that thanks alot
